### DISCUSSION TITLE
Simulation (quickstart) Feedback thread

### DISCUSSION THREAD

**Discussion by schmittlema** [prl-mushr/mushr#86] [Software]
This thread is for feedback and questions about the simulation/quickstart tutorial

**Comment by Lilly-yang** [comment]
Hi Matt,
I tried to change the map but it failed. Even if I change the argument of the `map_sever` group from `<arg name="map" value="$(eval _env_map if _env_map else map)" />` to `<arg name="map" value="$(eval map)" />`, it still doesn't work.
![image](https://user-images.githubusercontent.com/39558023/191059005-6a7023f3-c24e-467c-bb75-0b1858f57038.png)
![image](https://user-images.githubusercontent.com/39558023/191060347-e109b982-7f7a-448f-91a4-ce358f32257b.png)
Did I do something wrong?
Thanks!

**Reply by Lilly-yang** [reply]
I know why, somehow my edits in VS Code are not saved to the original file. Excuse me...
Should I delete this comment?
